### PR TITLE
[DW-291] Implement Cookiebot

### PR DESCRIPTION
**Pull request by robValtech**

### CODE REVIEW

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/SitePage.java [hunk 1]
```diff
@@ -104,10 +104,16 @@ public WebElement findPageElement(String elementName, int nth) {
...
 
     public void clickCookieAcceptButton() {
-        WebElement cookieAcceptButton = helper.findOptionalElement(By.id("cookieAcceptButton"));
+        WebElement cookieAcceptButton = helper.findOptionalElement(By.id("CybotCookiebotDialogBodyButtonAccept"));
         if (cookieAcceptButton != null
             && cookieAcceptButton.isDisplayed()) {
             cookieAcceptButton.click();
+
+            try {
+                Thread.sleep(1000);
```
Comment (jacsamell) [inline acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/SitePage.java]: Is this just for testing purposes?

File: acceptance-tests/src/test/resources/features/site/search.feature [hunk 2]
```diff
@@ -147,6 +147,7 @@ Feature: Basic search
 
     Scenario: Clicking on the 'Data and Information' tab displays the results page with publications, data sets, series and methodologies
         Given I navigate to the "search" page
+
```
Comment (jacsamell) [inline acceptance-tests/src/test/resources/features/site/search.feature]: ?

**Comment by ngta1** [comment]
Looks good.  Just the same comments that @jacsamell has already raised.

**Comment by robValtech** [comment]
Thank you guys!

**Pull request merged**